### ISSUE TITLE
Postgres connector cannot resume if no events in CDC stream received

### ISSUE THREAD

**Issue by karolisg** [getdozer/dozer#2287]
This bug happens when we don't receive any message in CDC stream after reading the initial snapshot of data in postgres database.
In this case, we have LSN of publication slot, which can be used for snapshotting done event.

https://link.excalidraw.com/readonly/cBRn1ElaavxaTJtJ6CvP
<img width="780" alt="Screenshot 2024-01-02 at 15 04 44" src="https://github.com/getdozer/dozer/assets/1695346/deaf2d0c-49f6-48c7-b65e-de13bbfdaca9">

**Comment by chubei** [comment]
Yes, we should send a `Some(OpIdentifier::new(lsn, MAGIC_NUMBER))` to make it resumable once snapshotting is done.
